docs/general/conflux-basics/glossary.md

```diff
@@ -98,6 +98,9 @@ ERC20 is a standard for tokens on the Ethereum blockchain. It specifies a set of
 ### **ERC721**
 ERC721 is a standard for non-fungible tokens (NFTs) on the Ethereum blockchain. Unlike ERC20 tokens, which are identical to each other, each ERC721 token is unique. This makes them suitable for representing ownership of unique items or assets. Like ERC20, ERC721 tokens can also exist on the Conflux network, especially if they are transferred from the Ethereum network.
 
+### **ERC1155**
+ERC1155 is a Ethereum token standard that supports multiple token types, including fungible and non-fungible tokens, within a single contract. This reduces transaction costs by enabling batch transfers and offers complex access controls. ERC1155 tokens can also exist on the Conflux network, enhancing flexibility and interoperability for digital assets in gaming, art, and beyond.
```

Comment:
```suggestion
ERC1155 is an Ethereum token standard that supports multiple token types, including fungible and non-fungible tokens, within a single contract. This reduces transaction costs by enabling batch transfers and offers complex access controls. ERC1155 tokens can also exist on the Conflux network, enhancing flexibility and interoperability for digital assets in gaming, art, and beyond.
```

Apart from the above change, would you explain why ERC1155 "offers complex access controls" ? I don't think the access control provided by ERC1155 is worth mentioning from my point of view

Reply:
The "complex access controls" part means that the ERC1155 standard supports advanced permission settings within a single contract. Developers can define granular permissions for different actions related to the token management and transfer, allowing for a more secure and customized token interaction experience. This is especially useful for creating a nuanced permission system in applications like digital asset libraries, gaming platforms, or decentralized marketplaces.


It's okay not to mention it